assets/src/blocks/Articles/ArticlesBlock.js

```diff
@@ -65,19 +66,26 @@ export class ArticlesBlock {
           },
         }
       ],
-      edit: withSelect(select => {
-        const postType = select('core/editor').getCurrentPostType();
-        const postId = select('core/editor').getCurrentPostId();
-        return { postType, postId };
-      })(({ isSelected, attributes, setAttributes, postType, postId }) => {
+      edit: ({isSelected, attributes, setAttributes}) => {
+
+        const {postType, postId} = useSelect((select) => ({
+            postType: select('core/editor').getCurrentPostType(),
+            postId: select('core/editor').getCurrentPostId()
+          })
+          , []);
+
+        const {posts, totalPosts} = useArticlesFetch(attributes, postType, postId);
+
         return <ArticlesEditor
-          attributes={attributes}
-          postType={postType}
-          setAttributes={setAttributes}
-          isSelected={isSelected}
-          postId={postId}
-        />
-      }),
+          attributes={ attributes }
+          setAttributes={ setAttributes }
+          isSelected={ isSelected }
+          postType={ postType }
+          postId={ postId }
```

Comment:
You pass down the `postId` here but I don't see it used anywhere in `ArticlesEditor` anymore...?